.circleci/config.yml

```diff
@@ -99,6 +98,7 @@ aliases:
             GENERATE_JEST_REPORT: "true"
             JEST_JUNIT_OUTPUT_DIR: ./test-results/jest-node/
             JEST_JUNIT_OUTPUT_NAME: results.xml
+            JEST_JUNIT_REPORT_TEST_SUITE_ERRORS: "true"
```

Comment:
> Reports test suites that failed to execute altogether as error. Note: since the suite name cannot be determined from files that fail to load, it will default to file path.

Doesn't hurt to enable that I guess

Reply:
Were you able to see improved reporting with `JEST_JUNIT_REPORT_TEST_SUITE_ERRORS` enabled? I looked at the errors in CI for this PR but can't tell if it's different

Reply:
No, didn't seem to make a difference really